src/Symfony/Component/Workflow/Event/Event.php

```diff
@@ -17,6 +17,8 @@
 use Symfony\Contracts\EventDispatcher\Event as BaseEvent;
 
 /**
+ * @template TSubject of object
```

Comment:
Given that the subject is only used in return types and not in argument types (not counting the constructor), this could use `@template-covariant` rather than `@template`, which is more flexible (same for child classes)

Reply:
covariant gives error, i replace it with `@template`, for now it's ok and as i see no errors anymore